### DISCUSSION TITLE
New knob und switch speeds

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#6629] [General]
Hi developer,

I guess the new animation speed for switsches, buttons and knobs is a bit to high ..... espacially the switches are really fast movibg ... 

Greets Klaus

**Comment by andrejetcat** [comment]
yes, I agree! To my feels, its to fast, too! I hope it could be switched back.

**Comment by hotshotp** [comment]
this is because you are experiencing a change from what you are used to. Irl the switch basically instantly snaps to its position - you push the switch and it jumps to its detent unless you purposefully grab the switch and slowly move it. this is a much requested change and it is much better now.
